standup_notes/__main__.py

```diff
@@ -24,7 +24,7 @@ def main():
     parser.add_argument('-l', '--list', help='List all stand-up notes.', action='store_true')
     parser.add_argument('-r', '--read', help='Read stand-up notes',
                         action='store_true')
-    parser.add_argument('-c', '--copy', help='Copy stand-up notes', action='store_true')
+    parser.add_argument('-c', '--copy', help='Copy stand-up notes to clipboard', action='store_true')
```

Comment:
I don't think this help string makes sense anymore, since it doesn't actually copy to clipboard if you have "edit" specified as well. Could you either expand on the help string to be more descriptive, or add another argument that's specifically for if you want to copy the previous days' notes?

Reply:
I have changed the help string

Reply:
I'm not certain it's super clear yet. It says it's used to either copy yesterday's notes or "call used to copy notes to clipboard". Could you specify how to use that functionality? Something like "If combined with the 'edit' flag, copies the previous day's TODO section, otherwise copies the specified day's notes to clipboard". It might not necessarily be "yesterday" if you include it when editing tomorrow's notes.